sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -98,10 +104,31 @@ if (MSVC)
   target_compile_options(azure-core-test PUBLIC /wd26495 /wd26812 /wd6326 /wd28204 /wd28020 /wd6330 /wd4389)
 endif()
 
+if (DISABLE_PROXY_TESTS)
+message(STATUS "Disabling proxy tests because they cannot be supported on this environment." )
+add_compile_definitions(DISABLE_PROXY_TESTS)
```

Comment:
Same nit here:
```suggestion
  message(STATUS "Disabling proxy tests because they cannot be supported on this environment." )
  add_compile_definitions(DISABLE_PROXY_TESTS)
```